src/dpp/message.cpp

```diff
@@ -1053,30 +1053,30 @@ message& message::fill_from_json(json* d, cache_policy_t cp) {
 	/* Fill in member record, cache uncached ones */
 	guild* g = find_guild(this->guild_id);
```

Comment:
This should be moved to be defined as part of the else if, on line `1065` as, if you have the cache off, you're still calling the cache method and then never using it.

Reply:
After conversations in discord, this will be resolved in a separate PR.